health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/appConfigurationRedesign/AppLocalisationWrapper.js

```
@@ -18,7 +19,8 @@
       if (state.find((item) => item.code === action.payload.code)) {
         return state;
       } else {
-        return [...state, { code: action.payload.code, en_IN: "", pt_IN: "", fr_IN: "" }];
+        const newEntry = action.payload.enabledModules.reduce((acc, locale) => ({ ...acc, [locale.value]: "" }), { code: action.payload.code });
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)